src/bot.ts

```diff
@@ -106,6 +111,11 @@ export default (app: Probot) => {
       context.log.info(`Sender didn't tag @gitpoap-bot explicitly in this comment`);
       return;
     }
+    // Check if organization is valid
+    if (!organization) {
+      context.log.error(`Organization is empty`);
```

Comment:
can you add information on the repo name to this log. 


& can you include `Sentry.setExtra(..);` to set more relevant information here? there are some examples of using this function in this file

Reply:
pretty much - the point here is to answer the question -> what information would be useful to see if i need to debug an issue / exception that is thrown here.